posts/share-cpp/index.qmd

```diff
@@ -230,11 +230,13 @@ cppcheck --std=c++14 --language=c++ --enable=warning,style --error-exitcode=1 in
 ```
 
 Furthermore, header code can also be tested independently of the R(cpp) code that eventually wraps it.
-This is easy to do using the [Catch2 testing framework](link WIP), which is [conveniently available using {testthat}](link WIP) --- this is an extensive topic for another post.
-
-<!-- From the docs for Rcpp Attr:  -->
-<!-- https://gallery.rcpp.org/articles/mixing-modules-and-export/ -->
+This is easy to do using the [Catch2 testing framework](https://github.com/catchorg/Catch2), which is [conveniently available using {testthat}](https://testthat.r-lib.org/reference/use_catch.html) --- this is an extensive topic for another post.
 
 ## Conclusion WIP
 
-<!-- Three sentences here. -->
+Overall then, developing an Rcpp-based package with C++ code sharing in mind takes some organisation, or even reorganisation, of the C++ codebase.
```

Comment:
I'd cut 'overall then', as conclusion heading already suggests this to reader (or shorten to just 'overall').

Reply:
Fixed in https://github.com/epiverse-trace/epiverse-trace.github.io/pull/34/commits/19f9fea49dfa48aa52d007cde29a79615be7d630